app/assets/javascripts/client/views/assessments/permissions/manage_permission.html

```diff
@@ -60,7 +60,7 @@
             <p class="info">
               <fieldset>
                 <label class="level_label">Current Permission Level: </label>
-                <select class="permission-level" name="level">
+                <select class="permission-level" name="level" ng-disabled="user.role === 'network_partner'">
```

Comment:
Our expectations were met as it is, this is your expectation. would locking down the API so it can not be used to switch a network partner into a participant be enough for you?